employees_info/src/main/java/com/teachmeskills/servlet/ChangeCredServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.teachmeskills.servlet;
+
+import com.teachmeskills.util.PostgresDriverManager;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@WebServlet("/change-login")
+public class ChangeCredServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {

```

Comment:
It should be performed with doPost method

Reply:
Same thing as the /delete. Need more info, pls(

Reply:
Fixed.